kube-kvm/provision-kube-host-kvm.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+# This script is intended to be run on a SLES box installed as a KVM host.
+# After OS installation, the following commands should also be run to set up directory
+# permissions and the default network (as root):
+# - usermod -aG libvirt root
+# - virsh net-define net-default.xml
+# - virsh net-start default
+# - ntpdate time.nist.gov
+# - systemctl enable ntpd
+# - systemctl start ntpd
+
+set -o errexit
+
+export KUBE_VM_NAME=${KUBE_VM_NAME:-$1}
+export KUBE_VM_IMAGE_NAME=${KUBE_VM_IMAGE_NAME:-scf-libvirt-v2.0.7}
```

Comment:
That requires cloning the SCF directory.. if we change directory permissions on https://cf-opensusefs2.s3.amazonaws.com/vagrant/ we could parse it out of the output there?

Reply:
Ah, I didn't realize you expected people to run this without a full checkout. Nevermind, then, if that's the case. (How do they get the script otherwise?)